### ISSUE TITLE
option for original version of Bloom

### ISSUE THREAD

**Issue by Voland44** [ThirteenAG/GTAIV.EFLC.FusionFix#517]
### What should happen

is it possible to add option for old version of Bloom like it was before 2.0? new Bloom version that was added in 2.0 seems broken and intensified. for example when you at stairs in Middle Park, Niko's legs looks transparent.

also, i would appreciate if anyone can say how to manually edit bloom into old version in current FusionFix releases.

### Checklist

- [x] Check the [latest git build](https://ci.appveyor.com/api/projects/ThirteenAG/gtaiv-eflc-fusionfix/artifacts/GTAIV.EFLC.FusionFix.zip) in case it's already implemented.
- [X] [Search for other requests](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/search?q=my+issue&type=issues) of the same feature.

**Comment by RecklessGlue540** [comment]
You can edit bloom manually inside visualsettings.dat:
https://gtaforums.com/topic/934545-fusionfix/?do=findComment&comment=1072267723

**Comment by RecklessGlue540** [comment]
That was fixed.

**Comment by Tomasak** [comment]
any pictures? Preferably with comparison to console.

**Comment by RecklessGlue540** [comment]
Have you checked Console Gamma? It should look roughly on par with consoles, with it on:
PC:
![GTAIV 7_22_2024 9_48_40 PM](https://github.com/user-attachments/assets/c76c4338-2489-4cbf-abc3-7740c94fffb1)

Console:
![Xenia-canary (canary_experimental@eeb96f91a on Jul 15 2024) _  545407F2 v5 0  GTA IV _Direct3D 12 - RTV_DSV_ 7_22_2024 9_50_25 PM](https://github.com/user-attachments/assets/639af554-58f4-4f18-addb-67bc190a16f5)

PC:
![GTAIV 7_22_2024 9_51_11 PM](https://github.com/user-attachments/assets/a1f058aa-b818-44c9-98d1-b76004fedca4)

Console:
![Xenia-canary (canary_experimental@eeb96f91a on Jul 15 2024) _  545407F2 v5 0  GTA IV _Direct3D 12 - RTV_DSV_ 7_22_2024 9_50_35 PM](https://github.com/user-attachments/assets/87828aef-dcb3-4f33-8b3c-4a6aba917e23)

Some things to note:
- Console Gamma isn't exactly fully accurate in the current builds, it's going to be way darker in the next update fwiw
- If you can't see the "transparent" edges, turn up your screen's settings. It mostly depends on display contrast ratio as well from what I'm seeing, moving these pictures between my around 1000:1 TV and 3000:1 Monitor...
- Bloom will also be improved in the next update as well, it will have two modes
- Don't bother with the ingame display sliders, they're not very good and mid at best.

**Comment by Parallellines0451** [comment]
There are two differences between console and PC bloom:
1. Console uses a max filter in the bloom pre-pass to boost the intensity / brightness a bit. PC lacks that, so it was restored.
2. PC can run at any resolution, but it doesn't scale the bloom correctly beyond 720p, so it becomes smaller the higher the resolution is. This was fixed.

If you compare both vanilla PC and console at 720p you'll see that both of them look roughly the same (aside from console being brighter due to point #1).

In short, that's how it's intended to look, and the small bloom at 1080p, 1440p or 4K in vanilla is a bug.

**Comment by Voland44** [comment]
Tbh i dont like console gamma, so i didnt bother to test this option on PC

Any insights on how Bloom would be improved in next update.

Btw, when i replaced current version with my previously installed 1.96 version(basically overwrite files) i found out that... Bloom not that bad anymore. Idk maybe this way Bloom remained same. Anyway, thx for the tip with visualsettings.dat i lowered bloom from 10 to 7, thats sweet spot for me for now.

**Comment by Voland44** [comment]
> In short, that's how it's intended to look, and the small bloom at 1080p, 1440p or 4K in vanilla is a bug.

Disagree. Devs did such bloom settings with strong console gamma and 720p in mind. Idk if its just bugged on PC but with medium gamma and FHD resolutions, bloom behavior should have been tweaked accordingly.
